diff -urN motor-2.17.13.org/share/Makefile.am motor-2.17.13/share/Makefile.am
--- motor-2.17.13.org/share/Makefile.am	Sat Jun  2 20:29:55 2001
+++ motor-2.17.13/share/Makefile.am	Sat Jun  2 20:47:31 2001
@@ -12,11 +12,11 @@
 	-if test -d templates; then rm -f tmpl.tar.gz; fi
 
 install-data-local: tmpl.tar.gz
-	tar zxvf tmpl.tar.gz -C $(datadir)/motor/
-	-find $(datadir)/motor/ -name CVS -type d -exec rm -rf {} \;
+	tar zxvf tmpl.tar.gz -C $(DESTDIR)$(datadir)/motor/
+	-find $(DESTDIR)$(datadir)/motor/ -name CVS -type d -exec rm -rf {} \;
 	if test -d templates; then rm -f tmpl.tar.gz; fi
 
 uninstall-local:
-	rm -rf $(datadir)/motor/templates
+	rm -rf $(DESTDIR)$(datadir)/motor/templates
 
 EXTRA_DIST = tmpl.tar.gz $(mshare_DATA)
diff -urN motor-2.17.13.org/src/Makefile.am motor-2.17.13/src/Makefile.am
--- motor-2.17.13.org/src/Makefile.am	Sat Jun  2 20:29:55 2001
+++ motor-2.17.13/src/Makefile.am	Sat Jun  2 20:39:45 2001
@@ -1,5 +1,5 @@
 INCLUDES = -I../include -I../ktools/include -I../parser/include -I$(top_srcdir)/intl
-CPPFLAGS = -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
+CPPFLAGS = @CPPFLAGS@ -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
 
 bin_PROGRAMS = motor
 
@@ -7,5 +7,5 @@
     lookout.cc compile.cc conf.cc cvs.cc debug.cc debugp.cc edit.cc grep.cc \
     folder.cc plist.cc tagbrowser.cc templ.cc tlist.cc
 
-motor_LDADD = -lcurses ../parser/libparser.a ../ktools/src/libkonst.a @INTLLIBS@ @LIBS@
-CXXFLAGS = -g
+motor_LDADD = -lncurses ../parser/libparser.a ../ktools/src/libkonst.a @INTLLIBS@ @LIBS@
+CXXFLAGS = @CXXFLAGS@ -g
