--- motor-3.2.0/src/manager/Makefile.am~	Sat Aug 25 01:27:56 2001
+++ motor-3.2.0/src/manager/Makefile.am	Fri Feb 22 10:05:04 2002
@@ -7,7 +7,7 @@
     -I$(top_srcdir)/src/configuration -I$(top_srcdir)/src \
     -I$(top_srcdir)/src/tagbrowser -I$(top_srcdir)/parser/include
 
-CPPFLAGS = -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
+CPPFLAGS = @CPPFLAGS@ -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
 
 libmanager_a_SOURCES = motormanager.cc
 
--- motor-3.2.0/src/configuration/Makefile.am~	Sat Aug 25 01:27:56 2001
+++ motor-3.2.0/src/configuration/Makefile.am	Fri Feb 22 10:04:54 2002
@@ -7,7 +7,7 @@
     -I$(top_srcdir)/src/ui -I$(top_srcdir)/src/tagbrowser -I$(top_srcdir)/src \
     -I$(top_srcdir)/parser/include
 
-CPPFLAGS = -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
+CPPFLAGS = @CPPFLAGS@ -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
 
 libconfiguration_a_SOURCES = motorconfiguration.cc extkey.cc
 
--- motor-3.2.0/src/Makefile.am~	Wed Oct 17 08:47:47 2001
+++ motor-3.2.0/src/Makefile.am	Fri Feb 22 10:04:44 2002
@@ -10,7 +10,7 @@
     -I$(top_srcdir)/src/tagbrowser -I$(top_srcdir)/src/vcs \
     -I$(top_srcdir)/parser/include
 
-CPPFLAGS = -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
+CPPFLAGS = @CPPFLAGS@ -DSHARE_DIR=\"$(pkgdatadir)\" -DLOCALE_DIR=\"$(datadir)/locale\"
 
 bin_PROGRAMS = motor
 motor_SOURCES = main.cc motormisc.cc
--- motor-3.2.0/share/Makefile.am~	Sat Aug 25 01:27:56 2001
+++ motor-3.2.0/share/Makefile.am	Fri Feb 22 10:03:07 2002
@@ -12,8 +12,8 @@
 	-if test -d templates; then rm -f tmpl.tar.gz; fi
 
 install-data-local: tmpl.tar.gz
-	tar zxvf tmpl.tar.gz -C $(datadir)/motor/
-	-find $(datadir)/motor/ -name CVS -type d -exec rm -rf {} \;
+	tar zxvf tmpl.tar.gz -C $(DESTDIR)$(datadir)/motor/
+	-find $(DESTDIR)$(datadir)/motor/ -name CVS -type d -exec rm -rf {} \;
 	if test -d templates; then rm -f tmpl.tar.gz; fi
 
 uninstall-local:
